aineko/cli/dream.py

```diff
@@ -2,49 +2,188 @@
 # SPDX-License-Identifier: Apache-2.0
 """A wrapper around API calls to the Aineko Dream API."""
 
+import time
+
 import click
 import requests
 
 
-@click.command()
-@click.argument("prompt")
-@click.argument("api-key")
-@click.option(
-    "-u",
-    "--url",
-    default="https://dream.ap-6d356ccd-bfba-4110-994a-fe164ab8bf77.aineko.app",
-    help="API url to use for the Aineko Dream API.",
-)
-def dream(prompt: str, api_key: str, url: str) -> None:
-    """Main function to generate an aineko project using Aineko Dream.\f
+def create_project_from_prompt(
+    api_key: str,
+    url: str,
+    prompt: str,
+) -> str:
+    """Create a project from a prompt.
 
     Args:
-        prompt: Prompt to generate a project from.
         api_key: API key to use for the Aineko Dream API.
         url: URL to use for the Aineko Dream API.
+        prompt: Prompt to generate a project from.
+
+    Returns:
+        request_id: Request ID for the project generation.
+
+    Raises:
+        click.ClickException: If there is an error generating the project.
     """
     try:
-        click.echo("Generating project from prompt...")
-
         r = requests.get(
-            url=f"{url}/create/",
+            url=f"{url}/create",
             params={"prompt": prompt},
             headers={"x-api-key": api_key},
-            timeout=300,
+            timeout=60,
         )
         click.echo(f"Received response code: {r.status_code}")
```

Comment:
Should this be deleted? looks like debugging to me

Reply:
Yes this is deleted. We no longer expect a single response to take a long time since we are making multiple requests, each returning the status immediately.